CHANGELOG.md

```diff
@@ -4,6 +4,169 @@ This changelog goes through the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.com).
 
+## [1.22.0](https://github.com/go-gitea/gitea/releases/tag/v1.22.0) - 2024-05-24
+
+Since there are too many pull requests, this changelog will ignore bugfixes,
+misc and etc changes.
```

Comment:
add a link to https://github.com/go-gitea/gitea/pulls?q=is%3Apr+milestone%3A1.22.0+is%3Amerged ?
Then user can check all PRs quickly if they want.

Reply:
80ec02a